xchange-kucoin/src/main/java/org/knowm/xchange/kucoin/KucoinTradeService.java

```diff
@@ -141,7 +141,7 @@ public UserTrades getTradeHistory(TradeHistoryParams tradeHistoryParams) throws
     String nextPageCursor = null;
     if (startTime != null && startTime >= cutoffHistOrdersMillis) {
       Pagination<TradeResponse> fills =
-          getKucoinFills(symbol, page, TRADE_HISTORIES_TO_FETCH, startTime, endTime);
+          getKucoinFills(symbol, null, page, TRADE_HISTORIES_TO_FETCH, startTime, endTime);
```

Comment:
how does changing this to null affect behavior?

Reply:
It is the same behaviour. It is not possible to get trades for a single order. If there is standard way I can implement it.
when you add something like: TradeHistoryParamOrderId I will implement it.